### PR TITLE
Rfb Event Class

### PR DESCRIPTION
**Pull request by Aniak5**
• Add new RFB event class
image
image

### CODE REVIEW

File: objects/framebuffer.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "caption": "Frame Buffer",
+  "name": "framebuffer",
```
Comment (rroupski) [inline objects/framebuffer.json:3]: If the object  caption is _Frame Buffer_ (2 words), then the object name should `frame_buffer`?

File: objects/server.json [hunk 2]
```diff
@@ -26,7 +26,13 @@
     },
     "uid": {
       "description": "The unique server identifier.",
-      "requirement": "required"
+      "requirement": "recommended"
     }
+  },
+  "constraints": {
```
Comment (rroupski) [inline objects/server.json:32]: I think the server `name` should be recommended and the contracts will be:
```
"constraints": {
   "at_least_one": [
       "name",
       "uid"
     ]
  }

```
The `protocol_ver` does not look as a server identifier, thus it should not be used in the constraints.

File: objects/vnc.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{
+  "caption": "VNC Authentication",
+  "name": "vnc_auth",
+  "extends": "object",
+  "description": "The Virtual Network Computing (VNC) authentication object describes the vnc authentication values.",
```
Comment (rroupski) [inline objects/vnc.json:5]: Use all-upper-case in "...describes the **VNC** authentication...".

**Review by rroupski — CHANGES_REQUESTED** [review]
Can you bump up the version number to 0.21.0?

File: objects/framebuffer.json (hunk 1 above)
Comment (Aniak5) [reply objects/framebuffer.json:3]: Sounds good!

File: objects/server.json (hunk 2 above)
Comment (Aniak5) [reply objects/server.json:32]: Good point.

**Review by rroupski — APPROVED** [review]

**Pull request merged**